README.md

```diff
@@ -488,6 +488,147 @@ map.put("count", 3);
 CodeBlock.builder().addNamed("I ate $count:L $food:L", map)
 ```
 
+#### Place Holder
+
+##### `$$` emits a dollar sign
+
+Use $$ to emit a dollar sign. It can add $ into the format string instead of being recognized as other Arguments.
+
+```JAVA
+CodeBlock.builder().add("Print dollar sign $$\nThe price of $L is $L$$","burger",10);
+```
+
+Result:
+
+```
+Print dollar sign $
+The price of burger is 10$
+```
+
+
+
+##### `$>` `$<` increases/decreases the indentation level.
+
+```java
+CodeBlock codeBlock =CodeBlock.builder().add("CodeBlock codeBlock =CodeBlock.builder()")
+                .add("$>$>")
+                .add("\n.add($S, $S)", '\'', "&#39;")
+                .add("\n.add($S, $S)", '>', "&gt;")
+                .add("\n.build()")
+                .add("$<$<")
+                .build();
+```
+
+```java
+CodeBlock codeBlock =CodeBlock.builder()
+    .add("'", "&#39;")
+    .add(">", "&gt;")
+    .build()
+```
+
+##### `$[` `$]` begin/end a statement. 
+
+`$[` and `$]` are used to start or end a statement. What should be mentioned is that every line after the first line will be double-indented if there is multiple statements.
+
+```java
+CodeBlock codeBlock =CodeBlock.builder().add("class Student")
+                .add("{$[\n")
```

Comment:
This is a very bad example: these are not statements!

Switch the examples for `$>` and `$[`.